### DISCUSSION TITLE
How to correctly hide data storage inputs for submitting information?

### DISCUSSION THREAD

**Discussion by GomezIvann** [dxc-technology/halstack-react#547] [Ideas] [answered]
Lately while searching I found some other implementations or opinions about how to hide an input that stores data, which go far from just doing `display: none`, as we currently have. I would like to discuss which way of doing this is better for screen-readers, fulfilling all conventions, etc.

1. Using [`<input type="hidden">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden).
2. `display: none` as we did until now but also adding `aria-hidden="true"` for screen-readers.
3. Using [html5 `hidden` attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-hidden-attribute).
4. Other ideas I didn't consider and would be interesting to discuss 😃.

**Comment by aweell** [comment]
Maybe we already had this conversation... but I thought we were currently using the second approach: `display:none` + `aria-hidden= "true"`.

**Reply by GomezIvann** [reply]
Yes! But after looking at other design systems I find more interesting the first option. It has every benefit of point 2 and it's is a specific HTML property for this kind of scenario.

**Comment by GomezIvann** [comment, marked as answer]
We have already followed the `<input type="hidden" />` approach I suggested, so we can close this!
